public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -3,134 +3,714 @@ info:
+      operationId: showContainerNode
+      description: Returns a ContainerNode object
+      produces:
+      - application/json
+      parameters:
+      - name: id
+        in: path
+        description: ID of the resource to return
+        required: true
+        type: string
+        format: uuid
+      responses:
+        200:
+          description: ContainerNode info
+          schema:
+            "$ref": "#/definitions/ContainerNode"
+        404:
+          description: Not found
+  "/container_projects":
+    get:
+      summary: List ContainerProjects
+      operationId: listContainerProjects
+      description: Returns an array of ContainerProject objects
+      produces:
+      - application/json
+      responses:
+        200:
+          description: ContainerProjects array
+          schema:
+            type: array
+            items:
+              "$ref": "#/definitions/ContainerProject"
+  "/container_projects/{id}":
+    get:
+      summary: Show a ContainerProject
+      operationId: showContainerProject
+      description: Returns a ContainerProject object
+      produces:
+      - application/json
+      parameters:
+      - name: id
+        in: path
+        description: ID of the resource to return
+        required: true
+        type: string
+        format: uuid
+      responses:
+        200:
+          description: ContainerProject info
+          schema:
+            "$ref": "#/definitions/ContainerProject"
+        404:
+          description: Not found
+  "/container_templates":
+    get:
+      summary: List ContainerTemplates
+      operationId: listContainerTemplates
+      description: Returns an array of ContainerTemplate objects
+      produces:
+      - application/json
+      responses:
+        200:
+          description: ContainerTemplates array
+          schema:
+            type: array
+            items:
+              "$ref": "#/definitions/ContainerTemplate"
+  "/container_templates/{id}":
+    get:
+      summary: Show a ContainerTemplate
+      operationId: showContainerTemplate
+      description: Returns a ContainerTemplate object
+      produces:
+      - application/json
+      parameters:
+      - name: id
+        in: path
+        description: ID of the resource to return
+        required: true
+        type: string
+        format: uuid
+      responses:
+        200:
+          description: ContainerTemplate info
+          schema:
+            "$ref": "#/definitions/ContainerTemplate"
+        404:
+          description: Not found
+  "/endpoints":
+    get:
+      summary: List Endpoints
+      operationId: listEndpoints
+      description: Returns an array of Endpoint objects
+      produces:
+      - application/json
+      responses:
+        200:
+          description: Endpoints array
+          schema:
+            type: array
+            items:
+              "$ref": "#/definitions/Endpoint"
+    post:
+      summary: Create a Endpoint
```

Comment:
This would read better if it were "Create an EndPoint". Same for the ones that follow.

Reply:
Why? The class is called `Endpoint` and "endpoint" is a single word, not two.

Reply:
Oh, "a" vs "an" should be fixed though. Didn't catch that one.

Reply:
Yeah, sorry for the confusion. I was referring to the "a" vs "an"

Reply:
Nice! I was able to fix the typo in my comment 🤓

Reply:
Fixed